charts/gsp-cluster/templates/02-gsp-system/service-operator/role.yaml

```diff
@@ -6,6 +6,16 @@ metadata:
   creationTimestamp: null
   name: {{ .Release.Name }}-service-operator-role
 rules:
+- resources:
```

Comment:
Is it possible I was supposed to update `config/rbac/role-not-patch.yaml` instead of add that kubebuilder:rbac comment earlier?